### ISSUE TITLE
Unclear guidance: Please update the system requirement in DESCRIPTION file

### ISSUE THREAD

**Issue by malcolmbarrett** [extendr/rextendr#104]
`use_extendr()` requests: `Please update the system requirement in DESCRIPTION file.` I think this could be a bit clearer. Is there some generally useful boilerplate, e.g. what is in rextendr's DESCRIPTION (`Rust 'cargo'; the crate 'libR-sys' must compile without error`), that `use_extendr()` could offer? 

In addition to offering clearer guidance here, we could suggest `desc` and modify it ourselves if installed, as usethis often does (desc is an implicit suggested pkg currently via devtools/usethis).

**Comment by clauswilke** [comment]
I'm not even sure what exactly should be in the system requirements. That's maybe the first thing to figure out.

**Comment by malcolmbarrett** [comment]
Yes, that makes sense. There's probably a related issue here with guiding users on their toolchain setup when rextendr can't successfully run. Something like `rust_sitrep()`, similar to `usethis::git_sitrep()`, that reports what rextendr can find and make suggestions about troubleshooting
